tools/replaceColumn/replaceColumn.xml

```diff
@@ -63,14 +81,26 @@ with open("output_file", 'w') as output:
                label="Which column should be replaced?" />
         <param name="skip_lines" type='integer' value='0' label="Skip this many starting lines" />
         <param name="delimiter" type="select" label="Delimited by">
-            <option value="" selected="True">Tab</option>
-            <option value=" ">Whitespace</option>
+            <option value="" selected="True">Whitespace</option>
+            <option value=" ">Tab</option>
             <option value=".">Dot</option>
             <option value=",">Comma</option>
             <option value="-">Dash</option>
             <option value="_">Underscore</option>
             <option value="|">Pipe</option>
         </param>
+        <param name="unknowns_strategy" type="select" label="When an unknown value is encountered">
+            <option value="skip" selected="True">Skip / Do not print</option>
+            <option value="print">Print without modification</option>
+            <option value="error">Exit with an error</option>
+        </param>
+        <param name="pass_comments" type='text' value='#' label="Pass lines commented with this character through unchanged" help="An empty string will cause a replacement attempt to be made on all lines.">
```

Comment:
This sentence sounds strange to me as non native :( `Pass lines commented with this character through unchanged`

Reply:
`An empty string will cause a replacement attempt to be made on all lines.` -> `Empty string means the line will be removed`?

Reply:
I'll try to reword "Pass lines...".

"An empty string..." -> means there won't be any commented lines expected in the file and thus, every line is subject to both the replacement and the strategy for how to handle things that don't match (i.e. skip (don't print), print (without replacement), or error.  Essentially, and empty string is the default behavior.  I'll see if I can re-word these statements to be clearer as well...